pkg/driver/node_server_test.go

```diff
@@ -115,10 +116,12 @@ func TestNodePublishVolume(t *testing.T) {
 		hotPlugger := &driver.FakeDiskHotPlugger{}
 		d.DiskHotPlugger = hotPlugger
 
+		targetPath := path.Join(t.TempDir(), "some-path")
+
 		_, err := d.NodePublishVolume(context.Background(), &csi.NodePublishVolumeRequest{
 			VolumeId:          "volume-1",
 			StagingTargetPath: "/mnt/my-target",
-			TargetPath:        "/var/lib/kubelet/some-path",
```

Comment:
> [!NOTE]
In a Kubernetes environment, the `/var/lib/kubelet/` directory is used, but when trying to use the same path in unit tests on Local and CI, a permission error occurs.
Since this function(`NodePublishVolume`) involves creating and mounting volumes, I used a temporary directory for test.